### DISCUSSION TITLE
Error! Using Horilla on Ubuntu

### DISCUSSION THREAD

**Discussion by sharukhkhan234** [horilla-opensource/horilla#208] [Q&A]
Hi!,
I want to notify the Horilla team regarding an error that is occurring while providing a username and password.
First I installed this software on the window and It's working perfectly, and I am exploring this software. Now I tried to install it on Linux(Ubuntu), It is successfully installed and also runs the server, but an error is occurring while providing the username and password, it generates this error, see below.
![Screenshot from 2024-06-12 14-41-49](https://github.com/horilla-opensource/horilla/assets/157000111/803f010a-532b-46a8-8e22-e234423b5b13)

**Comment by horilla-opensource** [comment]
> Hi!, I want to notify the Horilla team regarding an error that is occurring while providing a username and password. First I installed this software on the window and It's working perfectly, and I am exploring this software. Now I tried to install it on Linux(Ubuntu), It is successfully installed and also runs the server, but an error is occurring while providing the username and password, it generates this error, see below. ![Screenshot from 2024-06-12 14-41-49](https://private-user-images.githubusercontent.com/157000111/338918200-803f010a-532b-46a8-8e22-e234423b5b13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxODgyNDEsIm5iZiI6MTcxODE4Nzk0MSwicGF0aCI6Ii8xNTcwMDAxMTEvMzM4OTE4MjAwLTgwM2YwMTBhLTUzMmItNDZhOC04ZTIyLWUyMzQ0MjNiNWIxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMlQxMDI1NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYTQwNDdmMDUzYWVhMjEyZjViZWNkNGJhMTdkOTkxM2YyMTQ1NTY0NDYyY2M0ZDc4YTczZjM3MjNiZjkwMWUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.F-ytCnAw5CNlxfQ1yQnYls9X_Gt2qCUKzCw_CaNmXUQ)



Hi @sharukhkhan234 ,
The issue is with the test database from the latest code base.
Will update now.

With Regards,
**Team Horilla**

**Reply by horilla-opensource** [reply]
Hi @sharukhkhan234 ,
The test database is updated in the git repo. Please pull the latest code and try again.

With Regards,
**Team Horilla**
